cmd/service-controller/bridges.go

```diff
@@ -476,7 +474,6 @@ func addIngressBridge(sb *ServiceBindings, siteId string, bridges *qdr.BridgeCon
 		case ProtocolTCP:
```

Comment:
Observation when adding listener via 'skupper service create tcp-go-echo 9090' router logs repeat following:

2022-03-23 17:04:22.369692 +0000 TCP_ADAPTOR (info) Configured TcpListener for tcp-go-echo:9090, 0.0.0.0:1024
2022-03-23 17:04:22.369904 +0000 TCP_ADAPTOR (notice) PN_LISTENER_OPEN Listening on 0.0.0.0:1024
2022-03-23 17:04:52.346521 +0000 TCP_ADAPTOR (info) Deleted TcpListener for tcp-go-echo:9090, 0.0.0.0:1024
2022-03-23 17:04:52.346571 +0000 TCP_ADAPTOR (info) Deleted TCP bridge configuation 'tcp-go-echo:9090' for address tcp-go-echo:9090, 0.0.0.0:1024, siteId 68f041bb-a772-45f5-8557-307b3ffb7cc2. Connections opened:0, closed:0. Bytes in:0, out:0
2022-03-23 17:04:52.348172 +0000 TCP_ADAPTOR (info) Configured TcpListener for tcp-go-echo:9090, 0.0.0.0:1024
2022-03-23 17:04:52.348403 +0000 TCP_ADAPTOR (notice) PN_LISTENER_OPEN Listening on 0.0.0.0:1024
2022-03-23 17:04:52.355215 +0000 TCP_ADAPTOR (info) Deleted TcpListener for tcp-go-echo:9090, 0.0.0.0:1024
2022-03-23 17:04:52.355250 +0000 TCP_ADAPTOR (info) Deleted TCP bridge configuation 'tcp-go-echo:9090' for address tcp-go-echo:9090, 0.0.0.0:1024, siteId 68f041bb-a772-45f5-8557-307b3ffb7cc2. Connections opened:0, closed:0. Bytes in:0, out:0
2022-03-23 17:04:52.357131 +0000 TCP_ADAPTOR (info) Configured TcpListener for tcp-go-echo:9090, 0.0.0.0:1024
2022-03-23 17:04:52.358054 +0000 TCP_ADAPTOR (notice) PN_LISTENER_OPEN Listening on 0.0.0.0:1024
2022-03-23 17:04:52.363323 +0000 TCP_ADAPTOR (info) Deleted TcpListener for tcp-go-echo:9090, 0.0.0.0:1024
2022-03-23 17:04:52.363373 +0000 TCP_ADAPTOR (info) Deleted TCP bridge configuation 'tcp-go-echo:9090' for address tcp-go-echo:9090, 0.0.0.0:1024, siteId 68f041bb-a772-45f5-8557-307b3ffb7cc2. Connections opened:0, closed:0. Bytes in:0, out:0

Reply:
Yes, that is because the host gets read back as 0.0.0.0, so the comparison fails. Am looking at a fix now.